### DISCUSSION TITLE
Feature to "pin" window on top of all other windows

### DISCUSSION THREAD

**Discussion by JGStyle** [rxhanson/Rectangle#280] [Ideas]
it would be soooooo amazing if you could pin any window via rectangle to stay on top of other windows, even if it is not the active app

**Comment by rxhanson** [comment]
At quick glance, this functionality might only be attainable by forcing a user to disable SIP. In other words, not something that would be added in to Rectangle. If someone did find a way to implement it without having to disable SIP, then I would consider a PR (closing this one out since I'm don't have plans to go down that path, sorry!)
